.travis.yml

```diff
@@ -1,49 +1,76 @@
-sudo: required
-
-language: generic
+sudo: 'required'
 
 services:
-  - docker
+  - 'docker'
+
+language:
+  - 'bash'
+
+env:
+  global:
+    - NODE_RED_VERSION=x.y.z
+    - TARGET=raymondmm/node-red
+    - QEMU_VERSION=v3.0.0
+  matrix:
+    - DOCKER_FILE=Dockerfile.alpine-tmpl BUILD_FROM=amd64/node NODE_VERSION=10 OS=alpine QEMU_ARCH=x86_64 ARCH=amd64
```

Comment:
Should we just do v10 ? or should we include v12 (for testing only ?)

Reply:
I'll like to purpose to merge the new PR first and build all images according the new way. If all works fine, we can easily adopt Node v12.